tests/phpunit/CRM/Contact/SelectorTest.php

```diff
@@ -176,7 +176,7 @@ public function testPrevNextCache() {
       'expected_query' => [
         0 => 'default',
         1 => 'default',
-        2 => "WHERE  ( civicrm_email.email LIKE '%mickey@mouseville.com%' )  AND ( 1 )",
+        2 => "WHERE  ( civicrm_email.email LIKE '%mickey@mouseville.com%' )  AND contact_a.is_deleted = 0",
```

Comment:
This is_deleted suggests the contact in the test does not have permission to view deleted contacts?

Reply:
Looks like the cacheClause function was not respecting the view deleted contacts permission